### ISSUE TITLE
Hardcoded path to joker fails on Apple M1 Silicon 

### ISSUE THREAD

**Issue by Dimentium** [yqrashawn/GokuRakuJoudo#109]
```
❯ goku
Exception in thread "main" java.io.IOException: Cannot run program "/usr/local/opt/joker/bin/joker": error=2, No such file or directory
```

cause of 

```
❯ which joker
/opt/homebrew/bin/joker
```

I found similar issues description

> According to https://docs.brew.sh/Installation, /usr/local is for Intel binaries, /opt/homebrew for ARM. 

here [stackexchange](https://apple.stackexchange.com/questions/410825/apple-silicon-port-all-homebrew-packages-under-usr-local-opt-to-opt-homebrew)

**Comment by yqrashawn** [comment]
what does `brew --prefix joker` show on your mac?

**Comment by Dimentium** [comment]
```
brew --prefix joker
/opt/homebrew/opt/joker
~
```

**Comment by yqrashawn** [comment]
Seems the project runs out of Travis CI credits. Circle CI doesn't provide free macos image. I need time to investigate cross compile with graalvm.

**Comment by MuhammedZakir** [comment]
Isn't ` shell/sh joker "--lint" path` better than hardcoding `joker`'s path? The current path does not take account of user installing joker with other means. E.g. downloading binary. Is there a reason for hardcoding path?

**Comment by yqrashawn** [comment]
There are problems getting the right PATH when launch gokuw with brew service , which using macos launchagents.

I just fixed it using homebrew formula api.

https://apple.stackexchange.com/questions/364973/how-can-i-use-home-or-environment-variable-in-plist-file-of-launchdaemons?noredirect=1&lq=1

**Comment by yqrashawn** [comment]
fixed in 0.3.7

**Comment by Dimentium** [comment]
works perfect!

**Comment by yqrashawn** [comment]
The release workflow with gh action is fixed in c93edd4ff194032de52461aa1e8d3dcbe51cb98a
